res/sections/45-WirelessStandards.tex

```diff
@@ -0,0 +1,348 @@
+    again, thus reducing the overall interference.
+  }
+  \label{fig:802.11ws:broadcast}
+\end{figure}
+
+The car movement needs to be kept in consideration: cars can move very fast,
+causing variable transmission range. On top of that, a car cannot be sure to
+be the farthest car receiving that broadcast message, as explained in
+Figure~\ref{fig:802.11ws:broadcast}. So who has to forward the alert message?
+It's important to pick up the right vehicle that has to propagate it.
+If the message has already been rebroadcast by a following vehicle do not
+forward it, because it could be redundant. A way to reduce the message
+retransmission is to use a sort of back-off mechanism, or otherwise set vehicles
+contention window with an inverse proportion of the distance from the sender,
+but it has the unrealistic assumption that there is an unique, constant, and
+known-a-priori transmission range.
+
+\subsubsection{Using minimum connect dominating set} The MCDS identify the
+minimum number of sets that allows you to cover the whole set\footnote{A minimum
+  connected dominating set of a graph G is a connected dominating set with the
+  smallest possible cardinality among all connected dominating sets of G. The
+  connected domination number of G is the number of vertices in the minimum
+  connected dominating set. See more at
+  \url{https://en.wikipedia.org/wiki/Connected_dominating_set}.
+}.
+Although it seems a good solution for this problem, it has the following
+issues:
+\begin{itemize}
+\item only the one in the MCDS can propagate the message, making the solution
+  optimal, but not feasible: I would need a central station then knows all the
+  other nodes position
+\item this solution is not considering the number of hop a message has to
+  traverse
+\end{itemize}
+
+\subsubsection{Urban multi-hop broadcasting protocols} This broadcasting
+protocols uses \textbf{jamming} signals to determine the next forwarder:
+vehicles receiving an alert message emit a jamming signal for a time that's
+proportional to the distance from the sender. The last one stops the jamming
+signal and forward the message.
+
+The problem of this solution is that is slow, and for this is not suitable for
+alert messages (jamming signal phase delays the transmission of the message).
+
+\subsubsection{Fast Broadcast}
+Fast broadcast is designed to have Alert Messages covering the area of interest
+in less time possible. It uses two type of packets: \textit{Hello messages} and
+\textit{Alert messages}. Hello messages contain the sender's position and the
+maximum forward distance from which another vehicle has been heard transmitting
+another hello message. In particular it contains:
+\begin{itemize}
+\item CMBR (\textit{Current Maximum Backward Range}): the maximum backward
+  distance at which some car would be able to hear me
+\item CMFR (\textit{Current Frontward Range}): the maximum distance from which
```

Comment:
Current-turn Maximum Back Range: the estimation of the maximum frontward  distance  from  which  another  car along  the  strip-shaped  area-of-interest  can  be  heard  by  the  considered  one.  

Current-turn Maximum Front Range: the estimation of the maximum backward distance at which the considered car can be heard.